src/main/java/com/samoilenko/onlinebookstore/exception/EntityNotFoundException.java

```diff
@@ -0,0 +1,7 @@
+package com.samoilenko.onlinebookstore.exception;
+
+public class EntityNotFoundException extends RuntimeException {
+    public EntityNotFoundException(String message) {
+        super(message);
+    }
+}
```

Comment:
maybe add `public EntityNotFoundException(String message, Throwable cause) {
        super(message, cause);
    }`
    and use `Throwable cause`